.teamcity/src/subprojects/build/core/CoreBuild.kt

```diff
@@ -30,6 +32,17 @@ class CoreBuild(private val osJdkEntry: OSJDKEntry) : BuildType({
         }
     }
 
+    failureConditions {
```

Comment:
It looks like it applied only to the JVM build. Could you make this for js and native as well?
(Like id done with `defaultBuildFeatures`)